src/main/java/com/umc/networkingService/domain/test/controller/TestController.java

```diff
@@ -32,4 +34,55 @@ public String example() {
     public String ignore() {
         return "무시되는 API";
     }
+
+    private final TestService tempService;
+
+    @GetMapping("/test")
```

Comment:
테스트 api이긴 한데 주석을 다는 것보다 swagger에 api에 대해 약간의 설명으로 바꾸는게 좋을 거 같습니다.
```suggestion
    @Operation(summary = "테스트 API", description = "테스트 문자열을 반환하는 API입니다.")
    @ApiResponse(responseCode = "200", description = "테스트 문자열을 성공적으로 반환")
    @GetMapping("/test")
```